### ISSUE TITLE
transit perception factor

### ISSUE THREAD

**Issue by LinSiyu-TL** [TransLinkForecasting/rtmdoc#48]
Current writeup says "Note that the boarding perception factor used for transfer boarding only. The model assumed there is no penalty for initial boarding. Current model only account perception impact for work purpose only." - this is outdated

1. boarding time of 1 minute/boarding is added to transit travel time
2. each boarding is perceived as 10 minutes (not an exception for initial boarding)
3. perception factor is applied to all purposes in transit assignment

Info from Ali Darwiche:
basically in mode choice you feed the model the real boardings (not peceived)
the perception comes from the coefficients of mode choice
for example IVTT coeff ~ -0.05 
boarding coeff ~ -0.5
